hack/ci-integration.sh

```diff
@@ -7,7 +7,7 @@ go run ./vendor/github.com/onsi/ginkgo/v2/ginkgo \
     -v \
     --timeout=115m \
     --grace-period=5m \
-    --fail-fast \
+    --fail-fast=false \
```

Comment:
Out of interest, why do we switch to false here?

Reply:
Now if one case failed, the left cases will be marked as `[INTERRUPTED]`, I change this to continue run other test cases even if a certain test fails.

Reply:
Are we sure that the failed test case won't interfere with the later test cases? It will definitely be cleaned up properly? 

I wouldn't want the failure to then impact the later tests somehow

Reply:
I ran the disruptive cases locally on azure versioned-installer-fully_private_cluster-proxy cluster, didn't find the failed case interfere with the later test cases, will check on aws and gcp as well.

Reply:
I run several times on aws/gcp, didn't find the failed case interfere with the later test cases. switch to false here looks safe.